### PR TITLE
Clarify some loading tips

### PR DESCRIPTION
**Pull request by Rhundas**
Some of the current loading tips are potentially misleading, or could be clarified further.

### CODE REVIEW

File: RogueTech Core/GameTips/general.txt [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-Mouseover the Cybereye in your mech's status panel to see your sensor rolls.
+Press V to cycle between available melee targets.
+Mouseover the cybereye in your mech's status panel to see your sensor rolls.
 The colored dots on the ground show different types of terrain.
-Ctrl+click on Ammunition counter to eject Ammo before Moving.
-CTRL-A toggles visibility of ECM bubbles.
-Shift-Click on the skills tab of a pilot in the Barracks to reset their skills; Requires Argo Training Module Upgrade.
+CTRL+Click lets you jettison ammo bins to avoid heat-related explosions!
+CTRL+A toggles visibility of ECM bubbles.
+If you have the Argo Training Module Upgradem you can Shift+Click on the skills tab of a pilot in the Barracks to reset their skills.
```
Comment (Janaszar) [inline RogueTech Core/GameTips/general.txt]: This requires the second training module upgrade, right?

```suggestion
If you have the second Argo Training Module Upgrade you can Shift+Click on the skills tab of a pilot in the Barracks to reset their skills.
```
At the very least, there's an `m` on Upgrade.

File: RogueTech Core/GameTips/general.txt [hunk 2]
```diff
@@ -26,39 +26,38 @@ Positioning is key. Take the high ground or dig in deep.
...
 Ammo explodes. Don't have too much of it. Also, don't run out of ammo. But you do you, chief.
-Untrained Mechwarriors are more likely to panic. Punch through their armor to make them reach for the Eject button. 
+Untrained Mechwarriors are more likely to panic. Punch through their armor to make them reach for the eject button. 
 Scare your enemy into bailing out! Destroying their allies, flanking, and reducing their offensive capability all lead to panic.
 The danger zones in a DropShip landing area are no joke, no matter what your fellow Mechwarriors say.
 Plan your lance according to the range & strategy you intend to use.
 A 'Mech needs an engine core to function. 
 An XL engine will save a lot of weight, but if you lose a Torso Side, you lose the 'Mech.
-Bring some streaks or pulse lasers to deal with those pesky Lights!
+Weapons with accuracy bonuses or evasion ignoring, like Pulse Lasers, are fantastic against fast-moving enemies.
```
Comment (Janaszar) [inline RogueTech Core/GameTips/general.txt]:
```suggestion
Weapons with accuracy bonuses or that can ignore evasion, such as Pulse Lasers, are fantastic against fast-moving enemies.
```

File: RogueTech Core/GameTips/general.txt [hunk 3]
```diff
@@ -26,39 +26,38 @@ Positioning is key. Take the high ground or dig in deep.
...
 Watch your heat. Riding the redline will slow you down or blow you sky high.
-Riding the redline is dangerous... but fun.
+Riding the redline is dangerous...but fun.
```
Comment (Janaszar) [inline RogueTech Core/GameTips/general.txt]:
```suggestion
Riding the redline is dangerous... but fun.
```
As far as I know, an ellipse with following text gets a space between the two.

File: RogueTech Core/GameTips/general.txt [hunk 4]
```diff
@@ -26,39 +26,38 @@ Positioning is key. Take the high ground or dig in deep.
...
-ECM protects friendlies and degrades enemy sensors within its bubble.
+ECM Shield makes you harder to hit. ECM Jamming makes enemies more likely to miss.
 Active Probes boost your sensor check, improving the information you see about enemies at longer ranges.
 Stealth makes you harder to detect. It also provides a range based defense as well as extra defense the less you move!
 Narcs attach to a unit, providing visibility beyond sensor range and increased accuracy! This is opposed by enemy ECM, and lasts for a short duration.
 Tags increase your sensor check for tagged units, but this bonus degrades as they move from the position they were originally tagged.
 IFF jammers add a penalty to enemy sensor check's against a unit, making it harder to detect.
-UAV's allow you to do a quick scan of the Battlefield and Sensor Lock every Nearby Enemy.
-Melee combat in Roguetech is dramatically more difficult to start with, but a specialized melee 'Mech is a nightmare.
+UAVs allow you to do a quick scan of the battlefield and sensor lock every nearby enemy.
```
Comment (Janaszar) [inline RogueTech Core/GameTips/general.txt]:
```suggestion
UAV's allow you to do a quick scan of the battlefield and sensor lock every nearby enemy.
```
Plurals of abbreviations/acronyms tend to use `'s` in my experience.  It's not wrong as-is, just uncommon/rare to me.

File: RogueTech Core/GameTips/general.txt [hunk 5]
```diff
@@ -26,39 +26,38 @@ Positioning is key. Take the high ground or dig in deep.
...
+Melee combat in RogueTech is dramatically more difficult to start with, but a specialized melee 'Mech is a nightmare.
 Railguns and Gauss rifles deal tremendous damage at equally tremendous ranges. Reach out and touch someone.
-Killing Rocket Launcher or RL units before they blow their load is tactically advisable.
+Use your sensors to check enemy loadouts! 'Mechs and vehicles with rocket launchers (RLs) can do a lot of damage.
```
Comment (Janaszar) [inline RogueTech Core/GameTips/general.txt]: Feels like the old one is fine, and this should be a new one?
Maybe
`Better sensor rolls allow you to to check enemy loadouts!  You'll want to know if that truck is carrying a fluid gun, or something more sinister.`

File: RogueTech Core/GameTips/general.txt [hunk 6]
```diff
@@ -26,39 +26,38 @@ Positioning is key. Take the high ground or dig in deep.
...
 Melee attacks against the side and rear of a target have an increased chance to hit.
-Overheating 'Mechs in Roguetech do not suffer structure damage, but their ammunition bins have a small chance to explode.
+Overheating 'Mechs in Roguetech do not suffer structure damage, but they take penalties to speed and aim. If you run hot enough, your ammo can explode, too!
```
Comment (Janaszar) [inline RogueTech Core/GameTips/general.txt]:
```suggestion
Overheating 'Mechs in Roguetech do not suffer structure damage, but they take penalties to speed and to-hit. If you run hot enough, you can shutdown, take an injury, and your ammo can explode too!
```
Aim just sounds weird?
Added a bit on what else can happen with overheating.

File: RogueTech Core/GameTips/general.txt [hunk 7]
```diff
@@ -67,8 +66,8 @@ After a forest tile has been burned, it offers no cover and negatively affects t
 90% to hit is still 10% to miss. Dice don't follow your expectations.
 Friendly Fire isn't friendly. Don't put your mechs in the line of fire.
 Pyrrhic victories are for idealists, you're a merc. If the repairs are going to cost more than you are getting paid you should have left already.
```
Comment (Janaszar) [inline RogueTech Core/GameTips/general.txt]:
```suggestion
Pyrrhic victories are for idealists; you're a merc. If the repairs are going to cost more than you are getting paid you should have left already.
```
The two clauses are of opposing intents.

**Review by Janaszar — COMMENTED** [review]
Was linked to this from the Discord, where it was requested that a native English speaker review it.  (I'm pretty sure I fit that, though I know I have some quirks.)

Went through and added comments/possible changes as I saw fit.

File: RogueTech Core/GameTips/general.txt (hunk 1 above)
Comment (Rhundas) [reply RogueTech Core/GameTips/general.txt]: i can type don't bully me :<

File: RogueTech Core/GameTips/general.txt (hunk 5 above)
Comment (Rhundas) [reply RogueTech Core/GameTips/general.txt]: I felt that the innuendo implied was a little stale, personally, but obviously it's not my work so if people want it kept then that's fine.

Comment (Rhundas) [reply RogueTech Core/GameTips/general.txt]: Reverted in commit https://github.com/BattletechModders/RogueTech/pull/4298/commits/3ccb47eec110b546d94250908361e9a526950a4e

File: RogueTech Core/GameTips/general.txt (hunk 3 above)
Comment (Rhundas) [reply RogueTech Core/GameTips/general.txt]: This depends entirely on the style guide employed for writing. I've reached out to HBS to see if their writers can comment on what style guide was used for HBS BattleTech. When I get a response, I'll adopt whatever it is that they're using.

In the meantime, I'll leave this as [UK (ONS) style](https://style.ons.gov.uk/house-style/ellipsis/) for internal consistency. See commit https://github.com/BattletechModders/RogueTech/pull/4298/commits/d87638adbfe4c438ad60df4330b00cbaa50427fe

**Comment by Rhundas** [comment]
This should be good for a final review now. The style guide from HBS will only be a minor change that can be added later.

**Comment by ceakay** [comment]
Space or no space before ellipses? Inconsistent fixes on this.

**Comment by Rhundas** [comment]
Thought I fixed them all. I'll do another pass.

**Pull request closed without merging**